cgi-bin/DW/Console/Command/BonusIcons.pm

```diff
@@ -85,27 +85,26 @@ sub execute {
         }
 
     } elsif ( $cmd eq 'xfer' ) {
-        my $target_u = LJ::load_user( $cmdarg );
+        my $destination_u = LJ::load_user( $cmdarg );
 
         return $self->error( 'Invalid target user.' )
-            unless $target_u;
+            unless $destination_u;
         return $self->error( 'E-mail addresses do not match.' )
-            unless $to_u->prop( 'email' ) eq $target_u->prop( 'email' ); 
+            unless $to_u->prop( 'email' ) eq $destination_u->prop( 'email' ); 
+        return $self->error( 'E-mail address is not confirmed.' )
+            unless $to_u->is_validated && $destination_u->is_validated;
```

Comment:
Nah, go for easy and just make it "one or more email addresses is not confirmed" (psst: email, not e-mail, by site style handbook) -- any error in this process means I'm going to have to dig into it more, so it's ok if it's just a "something is broken" abort.

Jen can confirm, but: in looking through the code for opening up that other bug (and thus refamiliarizing myself with things), I think the 'unless' comparisons should maybe be done as "$to_u->email_raw" instead of "$to_u->prop( 'email' )".